DependencyInjection/App/Presenters/AnotherPresenter.php

```diff
@@ -0,0 +1,37 @@
+<?php declare(strict_types=1);
+
+namespace App\Presenters;
+
+use App;
+use Vendor;
+
+class AnotherPresenter extends Vendor\Presenters\SomePresenter
+{
+	/**
```

Comment:
Jednou jsme řešili víceřadkové dokumentační komentáře. Jsou potřeba jen u metod, nikde jinde nedávají význam. To si bude pamatovat @vitkutny. Nevznikl z toho nějaký závěr?